git_guide.md

```diff
@@ -0,0 +1,22 @@
+# Git guide
+This guide is designed to help you get started with git and use its basics commands.
+
+- [Getting started](#getting-started)
+- [First contribution](#first-contribution)
+- [Good first bugs](#good-first-bugs)
+
+## Getting started
+To get started with git,first install git from the given link or if you have already installed it then update to its latest version.
+- [Install git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+
+After installing git successfully, make a account at github.
+- [Get started with GitHub](https://help.github.com/en/github/getting-started-with-github/signing-up-for-a-new-github-account)
```

Comment:
This links to help guide that tells the user to "Go to GitHub's Pricing page." and "Read the information about the different products and subscriptions that GitHub offers, then click the upgrade button under the subscription you'd like to choose.": we probably just want to send them to the "create free account" page to keep things simple.